tf/scripts/backend_user_data.sh

```
@@ -13,8 +13,19 @@ unzip /home/ec2-user/backend.zip -d /home/ec2-user/
+# Ejecutar el contenedor Docker con variables para APM
+sudo docker run -d -p 8080:8080 \
+  -e DD_AGENT_HOST=172.17.0.1 \
+  -e DD_ENV=production \
+  -e DD_SERVICE=lti-backend \
+  -e DD_VERSION=1.0.0 \
+  lti-backend
+
+# -----------------------------------
+# Instalación del Agente de Datadog
+# -----------------------------------
+DD_API_KEY="${datadog_api_key}" DD_SITE="datadoghq.com" DD_APM_ENABLED=true DD_APM_NON_LOCAL_TRAFFIC=true bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
```

Shellcheck (0.11.0)

[warning] datadog_api_key is referenced but not assigned.

(SC2154)